schema.json

```diff
@@ -361,6 +361,36 @@
       "required": ["offset", "value", "variable"],
       "type": "object"
     },
+    "note": {
+      "additionalProperties": false,
+      "description": "Note object used to represent a SEQGEN note construct. Used by SEQGEN to output strings in the simulation results at specific times.",
```

Comment:
I'd recommend not tying this documentation to SEQGEN, even though some seq.json features were inspired by SEQGEN. Just describe what `note` is intended to do. The seq.json format will support the new seq2.0 tools, which will presumably replace SEQGEN at some point.

Reply:
Good point, done.

Reply:
Let me know if the new documentation is satisfactory @parkerabercrombie !